src/items/book.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'item'
+
+class Book < Item
+  attr_accessor :publisher, :title
+
+  def initialize(title, publisher, cover_state)
+    super()
```

Comment:
The initialize method needs to receive the publish_date argument as per the parent Item class to then pass it on to the super() constructor method.